src/Features/CSharp/Portable/ConvertToRecord/ConvertToRecordCommon.cs

```diff
@@ -394,6 +426,81 @@ private static SyntaxList<AttributeListSyntax> GetModifiedAttributeListsForPrope
             }));
         }
 
+        private static async Task RefactorInitializersAsync(
+            INamedTypeSymbol type,
+            SolutionEditor solutionEditor,
+            ImmutableArray<IPropertySymbol> positionalParameters,
+            CancellationToken cancellationToken)
+        {
+            var symbolReferences = await SymbolFinder
+                .FindReferencesAsync(type, solutionEditor.OriginalSolution, cancellationToken).ConfigureAwait(false);
+            var referenceLocations = symbolReferences.SelectMany(reference => reference.Locations);
+            var projectLookup = referenceLocations.ToLookup(refLoc => refLoc.Document.Project.Id);
```

Comment:
Oh, or is this needed because we might have a record syntax being updated while it also has a use inside, and the original solution here is from before the refactoring started at all?